media/css/sandstone/sandstone.css

```diff
@@ -515,42 +515,42 @@ input.focus {
 .billboard h6,
 .billboard .huge,
 .billboard .large {
-    color: #484848;
+  color: #484848;
 }
 
 nav.menu-bar {
```

Comment:
`nav.menu-bar` could likely be simplified to `.menu-bar`. I don't see any specificity issues here.